src/nodes/text.ts

```diff
@@ -1,14 +1,17 @@
-import NodeType from './type';
-import Node from './node';
+import { decode } from 'he';
 import HTMLElement from './html';
+import Node from './node';
+import NodeType from './type';
+
 
 /**
  * TextNode to contain a text element in DOM tree.
  * @param {string} value [description]
  */
 export default class TextNode extends Node {
-	public constructor(public rawText: string, parentNode: HTMLElement) {
+	public constructor(rawText: string, parentNode: HTMLElement) {
 		super(parentNode);
+		this._rawText = rawText;
 	}
```

Comment:
This change allows us to automatically reset trimmed text caches on update. I added the caching in a previous PR. This change prevents a future bug report with existing code.